javascript-loops-practice/odd-or-even.js

```diff
@@ -0,0 +1,9 @@
+/* exported oddOrEven */
+function oddOrEven(numbers) {
+  const results = [];
+  for (let i = 0; i < numbers.length; i++) {
+    if ((numbers[i] % 2) === 0) { results.push('even'); } else {
+      results.push('odd');
+    }
+  } return results;
+}
```

Comment:
This works, but you should really adjust the formatting. This is super hard to read. Anytime you use a bracket, you should move down into a new line.